test/keychange_listener_test.js

```diff
@@ -1,5 +1,6 @@
 describe('KeyChangeListener', function() {
   var phoneNumberWithKeyChange = '+13016886524';  // nsa
+  var address = new libsignal.SignalProtocolAddress(phoneNumberWithKeyChange, 1);
```

Comment:
What crash? This isn't used anywhere else in the file as of this rev. I added the same line in session-archive though, so maybe this is somehow an artifact of that?